### PR TITLE
update the managedclusterset concept

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: content/en/concepts/managedclusterset.md [hunk 1]
```diff
@@ -15,21 +15,21 @@ The `ManagedClusterSet` and `ManagedClusterSetBinding` API v1alpha1 version will
 
 The `ManagedClusterSet` and `ManagedClusterSetBinding` API v1beta1 version will be deprecated in OCM v0.9.0.
 
-For Hosted Mode, The `ManagedClusterSet` and `ManagedClusterSetBinding` API only support v1beta1 in OCM v0.9.0
+For Hosted Mode, The `ManagedClusterSet` and `ManagedClusterSetBinding` API only support v1beta1 in OCM v0.9.0.
```
Comment (qiujian16) [inline content/en/concepts/managedclusterset.md:18]: @ldpliu we should update this part now?

Comment (ldpliu) [reply content/en/concepts/managedclusterset.md:18]: We could support v1beta2 for hosted mode in **latest** code(0.10.0). 
For next release(0.10.0), we should remove it. I am not sure if it's ok to remove it now?

Comment (qiujian16) [reply content/en/concepts/managedclusterset.md:18]: I think we can remove it now

**Pull request merged**